app/domain/operations/products/import_rating_area.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+module Operations
+  module Products
+    # This class is to load rating areas.
+    class ImportRatingArea
+      include Dry::Monads[:result, :do]
+
+      def call(params)
+        values  = yield validate(params)
+        data    = yield load_data(values[:file])
+        _load = yield import_records(data, values[:year], values[:import_timestamp])
+        Success()
+      end
+
+      private
+
+      def validate(params)
+        return Failure('Missing File') if params[:file].blank?
+        return Failure('Missing Year') if params[:year].blank?
+        return Failure('Missing Import TimeStamp') if params[:import_timestamp].blank?
+        Success(params)
+      end
```

Comment:
Validate should also check to make sure that the county, zip, or county/zip pairs provided in the rating area information actually exist.  If they don't exist, as it is now, you will only get an error saying "Unable to import Rating Areas from file".

It also won't tell you which rating area is problematic, nor which pair or row is causing the problem.

Reply:
passing it off as we discussed